mandatory/3-function-output.js

```diff
@@ -1,18 +1,30 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+// This 'getNumber' function will return a random number between 0 (inclusive) and 10 (exclusive), which cannot be chosen or reset by the user.
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
+// This 's' function with the .concat() method is used to merge two or more arrays, by returning a new array and not changing the existing arrays.
 function s(w1, w2) {
   return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  let myFirstWord = firstWord + " ";
+  let mySecondWord = secondWord + " ";
+  let myThirdWord = thirdWord;
+  return myFirstWord.concat(mySecondWord, myThirdWord);
```

Comment:
Good effort on this Laura. Can you please think of a way to change this solution into a one-liner that is easy to read? Hint: use one only 1 method of concatenating strings. Let me know if you need further details on this

Reply:
thank you for the suggestion